docs/articles/configs/powerplans.md

```diff
@@ -0,0 +1,19 @@
+---
+uid: docs.powerplans
+name: Power Plans
+---
+
+# Power Plans
+
+This description concerns only version v0.11.2 and above of BenchmarkDotNet. In the previous versions, benchmarks in the power plans.
```

Comment:
We don't need a note about the BenchmarkDotNet version. The documentation is always relevant only for the latest version of BenchmarkDotNet.